recipes/selectolax/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "selectolax" %}
+{% set version = "0.3.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/selectolax-{{ version }}.tar.gz
+  sha256: d71d7ae7bec6e856b83111f270bc783e531d86e02ad3a78a37d8591ff8f3620f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - pip
+    - python  # [py>34]
+  run:
+    - python  # [py>34]
```

Comment:
What is this supposed to do? Only add Python as a dependency if the Python version is 3.5 or above? It's a bit weird. Maybe you meant `python >3.4`, but in that case is also not needed because conda-forge only builds Python 3.7 and above for a while.

Reply:
I see. I wanted to specify that the library supports `python>=3.5`. Is there a way to say that?

Reply:
Since you're not on `noarch`, the package will only be built for the python versions conda-forge currently supports (3.7-3.10 currently). If you need to restrict those, you'll need to use `skip` and then skip the python versions.

Reply:
Made changes.